theano/tensor/tests/test_opt.py

```diff
@@ -2,8 +2,6 @@
 
```

Comment:
Why do you remove the file theano/tensor/tests/shape_opt_cycle.pkl? You also removed another pickled file.